docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,15 +506,27 @@ If you submit a ledger API command (which can have multiple Daml commands), then
 
 Just putting all Daml commands into a single batch command does not help, as you might still create lots of views.
 
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the participants
+that host their informees are not a subset of their parents' informee participants. Therefore, in order to
+minimize the number of views, you should try to minimize either the number of changes of informees or make sure
+most of your actions share the same pool of participants that must be informed. Another alternative is to
+make your higher level action nodes include as much participants as possible so that subsequent action nodes in the tree
+are aggregated in the initial view, since these children nodes will only target a subset of those participants.
+Informees are roughly the signatories, observers of the contract and of the choice, and the controllers of the choice.
```

Comment:
They are defined in the ledger model's privacy section. I don't think it's useful to give a rough definition if you really want to optimize something.